cmd/create/oidcconfig/cmd.go

```diff
@@ -112,21 +113,58 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
+	oidcConfigInput := buildOidcConfigInput(r)
 	oidcConfigStrategy, err := getOidcConfigStrategy(mode)
 	if err != nil {
 		r.Reporter.Errorf("%s", err)
 		os.Exit(1)
 	}
-	bucketUrl, privateKeyFilename := oidcConfigStrategy.execute(r)
+	oidcConfigStrategy.execute(r, oidcConfigInput)
 	if r.Reporter.IsTerminal() {
 		r.Reporter.Infof("To create a cluster with this oidc config, please run:\n"+
 			"rosa create cluster --sts --oidc-endpoint-url %s "+
-			"--bound-service-account-signing-key-path ./%s", bucketUrl, privateKeyFilename)
+			"--bound-service-account-signing-key-secret-name %s",
+			oidcConfigInput.BucketUrl, oidcConfigInput.BucketName)
+	}
+}
+
+type OidcConfigInput struct {
+	BucketName         string
+	BucketUrl          string
+	PrivateKey         []byte
+	PrivateKeyFilename string
+	DiscoveryDocument  string
+	Jwks               []byte
+}
+
+func buildOidcConfigInput(r *rosa.Runtime) OidcConfigInput {
+	randomLabel := helper.RandomLabel(DEFAULT_LENGTH_RANDOM_LABEL)
+	bucketName := fmt.Sprintf("oidc-%s", randomLabel)
```

Comment:
This will result in a confusing SSM secret name, should we create a separate field for the SSM secret name? What do you think about naming this something like `rosa-oidc-private-key-%s`

Reply:
Sure, that works out